src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+    <a href="https://mashable.com/article/how-to-set-up-google-authenticator">
+      Google Authenticator
+    </a>
+     or <a href="https://authy.com/">Authy</a>.
+  </div>
+</InfoBanner>
+
+Watch more on the 2FA:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/m8jlnZuV1i4?start=3479&end=3875" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Uninstall browser extentions
+
+Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.
+
+#### Stay safe by:
+
+- Only install browser extensions from trusted sources
+- Removing unused browser extensions
+- Install Chrome extensions locally to stop auto-updating (Advanced)
+
+[More on the risks of browser extentions](<[https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)>)
+
+<Divider />
+
+## Crypto Security 101
+
+### Upgrade Your Knowledge
+
+One of the biggest reasons people get scammed in crypto generally is a lack of understanding. For example, if you don't understand that the Ethereum network is decentralized and owned by no one, then it's easy to fall prey to someone pretending to be a customer service agent that promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.
+
+<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />
+
+<DocLink to="/eth/" title="What is ether?" />
+<Divider />
+## Wallet Security
+
+### Don't give out your Private keys
+
+**Never, for anyone reason, hand out your private keys!**
+
+The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
+
+### Use a Hardware Wallet
+
+A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.
+
+Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.
+
+#### Try a hardware wallet:
+
+- [Ledger](<[https://www.ledger.com/](https://www.ledger.com/)>)
+- [Trezor](<[https://trezor.io/](https://trezor.io/)>)
+
+### Double check transactions
+
+Accidently sending crypto to the wrong wallet address is a common mistake. Unless you know the address owner, there will be no way for you to retrieve your funds.
```

Comment:
Perhaps worth mentioning that transaction on Ethereum (such as sending ETH) are irreversible?